projects/jetstack/cert-manager/README.md

```diff
@@ -17,3 +17,22 @@ You can find the latest versions of these images on ECR Public Gallery.
 [cert-manager Controller](https://gallery.ecr.aws/eks-anywhere/jetstack/cert-manager-controller) | 
 [CA injector](https://gallery.ecr.aws/eks-anywhere/jetstack/cert-manager-cainjector) | 
 [cert-manager Webhook Server](https://gallery.ecr.aws/eks-anywhere/jetstack/cert-manager-webhook)
+
+### Updating
+
+1. Update cert-manager tag when updating cluster-api tag if cluster-api is using a newer tag.
+   Use the same tag that cluster-api uses by default. For instance [cluster-api v1.0.1 uses cert-manager v1.5.3 by default]
+   (https://github.com/kubernetes-sigs/cluster-api/blob/v1.0.1/cmd/clusterctl/client/config/cert_manager_client.go#L30) so when updating
+   to cluster-api tag v1.0.1, update cert-manager tag to v1.5.3
+1. Review releases and changelogs in upstream [repo](https://github.com/jetstack/cert-manager) and decide on new version.
```

Comment:
Want to use real github ids?  And you can take me out...

Reply:
oh yes, switched to real github ids (not removing you!)

Reply:
😆